docs/ui/components/ContentSpotlight/index.tsx

```diff
@@ -19,22 +19,37 @@ type ContentSpotlightProps = {
   loop?: boolean;
   className?: string;
   containerClassName?: string;
+  playerWidth?: string | number;
+  playerHeight?: string | number;
+  autoplayYT?: boolean;
 };
 
 export function ContentSpotlight({
   alt,
   src,
   file,
   caption,
-  controls,
-  loop = true,
+  controls = true,
+  loop = false,
   className,
   containerClassName,
+  playerWidth,
+  playerHeight,
+  autoplayYT = true,
 }: ContentSpotlightProps) {
   const [forceShowControls, setForceShowControls] = useState<boolean>();
-  const isVideo = !!file;
+  const resolvedPlayerWidth = playerWidth ?? PLAYER_WIDTH;
+  const resolvedPlayerHeight = playerHeight ?? PLAYER_HEIGHT;
+  const hasCustomPlayerSize =
+    typeof playerWidth !== 'undefined' || typeof playerHeight !== 'undefined';
+  const getDimensionValue = (value: string | number) =>
+    `${value}${typeof value === 'number' ? 'px' : ''}`;
+  const videoUrl = file && (/^https?:\/\/|^\/\//.test(file) ? file : `/static/videos/${file}`);
   const playerRef = useRef(null);
   const isInView = useInView(playerRef);
+  const isVideo = !!videoUrl;
+  const isYoutubeVideo = typeof videoUrl === 'string' && videoUrl.includes('youtube.com/watch?v=');
```

Comment:
I'm concerned that this could be a bit error prone. There are many other ways to link to a youtube link, `https://youtu.be/<id>` for one.

I think we'd be better off passing in just the ID, same as in `VideoBoxLink`.